src/aa-classes/aaAutorecFunctions.js

```diff
@@ -10,6 +10,14 @@ export class AAAutorecFunctions {
         return newName;
     }
 
+    static getRealName(name, item) {
+        if(!name || !item) { return; }
+
+        if(!game.modules.has('babele')) return name;
+
+        return item.flags.babele.originalName || name;
```

Comment:
Use `getFlag` or at least use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to avoid breaking on items that do not have any babele translations at all